### ISSUE TITLE
DPL-948 - As a long read user (Craig) I would like to automatically populate the DNA control complex box barcode based on the binding kit version to save time and effort adding it manually

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-ui#1373]
**User story**
As a long read user (Craig) I would like to automatically populate the DNA control complex box barcode based on the binding kit version to save time and effort adding it manually

**Who are the primary contacts for this story**
Craig / Steve

**Who is the nominated tester for UAT**
Craig / Long Read Team

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] When a binding kit box barcode is scanned into the default well attributes for a v12 Sequel IIe run it will retrieve the correct DNA control complex box barcode
- [ ] When a binding kit box barcode is scanned into the default well attributes for a v12 Sequel IIe run it will populate the DNA control complex box barcode
- [ ] A valid list of Binding Kits and DNA control complex box barcodes

**Dependencies**
This story is blocked by the following dependencies:
- #1372
- sanger/traction-service#1129

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
- We will need a list of Binding Kits and the associated DNA Control Complex box barcodes
- As Binding Kit Box Barcode is a well attribute and DNA Control Complex box barcodes the user will need to use the default well attributes. This will need to be a change in process.
- This will be a tricky story as we are adding behaviour to an input box based on the SMRT Link Version and System.

**Comment by stevieing** [comment]
Closed as no longer needed.
